### ISSUE TITLE
Why Yeoman?

### ISSUE THREAD

**Issue by williamdclt** [theodo/theodo-stack-generator#160]
Hi guys!
We're having great results at TheodoUK with our own generator, and are now working at making the generator as easy to use as possible: I want it to be impossible to fail if you follow the docs.

Our main problem with that right now is that we rely on `django-admin` being installed globally (that was our MVP template engine) and that's quite a big moving piece. I've looked into a few template engine alternatives, amongst which Yeoman.

I'd like to know why you went for this engine rather than something else (Plop, Cookiecutter, Slush...), is there a particular reason?

**Comment by benja-M-1** [comment]
Great news for your generator. I regret that we didn’t manage to merge the two in one. I feel bad because we are reinventing... 

Clement started the generator with Yeoman because, I guess :
- he is a big fan of nodejs
- it was the one BAM chose too when he was there

We also have an issue with the global installation of php. For the moment we didn’t tacle it. But AFAIK there is no relation with Yeoman. The problem is that we must run commands in the vagrant box and not on the host to not rely on the host PHP version. And this is doable via the vagrant command line or docker. So there is no problem for us.

**Comment by williamdclt** [comment]
Yeah merging the 2 generators is not on my priorities right now but that might come in a later phase when we've managed to get below our 45min target

I'm not sure we'll go for Yeoman, Plop seems a better fit for us, we have a meeting tomorrow to decide on that :) If you're interested, my data sheet: https://docs.google.com/spreadsheets/d/1Ttzg-yzxPrZWa-91O7jL5kdeHsCW1nKMZFfg7jSf4qY/edit?usp=sharing

Thanks for the infos!
